### PR TITLE
Release v2.3.0

### PR DESCRIPTION
**Pull request by gsnider2195**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cookiecutter-nautobot-app"
-version = "1.0.0"
+version = "2.3.0"
```
Comment (gsnider2195) [inline pyproject.toml:3]: Was there a reason the version number wasn't being bumped in the pyproject.toml?

Comment (bryanculver) [reply pyproject.toml:3]: Let's change the mode: https://python-poetry.org/docs/basic-usage/#operating-modes

File: pyproject.toml [hunk 2]
```diff
@@ -17,48 +17,25 @@ pyyaml = "*"
 
 [tool.poetry.group.dev.dependencies]
 bandit = "*"
-black = "*"
-coverage = "^7.3.2"
+coverage = "*"
```
Comment (gsnider2195) [inline pyproject.toml:20]: I think we need to pin coverage for the same reason we pinned it in core:
https://github.com/nautobot/nautobot/blob/8dbbdaa307c90a5e315257c1d4f866c5b0f263af/pyproject.toml#L217

Newer versions cause problems with pdb breakpoints

Comment (gsnider2195) [reply pyproject.toml:20]: Will follow up later https://github.com/nautobot/cookiecutter-nautobot-app/issues/136

File: docs/admin/release_notes/version_2.3.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# v2.3 Release Notes
+
+This document describes all new features and changes in the release `2.3`. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v2.3.0] - 2024-08-06
+
+### Changed
+
+- [#112](https://github.com/nautobot/cookiecutter-nautobot-app/pull/112) - Replaced `mkdocs-version-annotations` mkdocs plugin with new `markdown-version-annotations` markdown extension.
```
Comment (cmsirbu) [inline docs/admin/release_notes/version_2.3.md:9]:
```suggestion
- [#112](https://github.com/nautobot/cookiecutter-nautobot-app/pull/112) - Replaced `mkdocs-version-annotations` mkdocs plugin with new `markdown-version-annotations` markdown extension. No change in functionality, it's the same package.
```

**Review by cmsirbu — APPROVED** [review]
LGTM! Left one clarification for the release notes in case anyone wonders about the replacement.

**Pull request merged**